servers/rendering_server.cpp

```diff
@@ -1321,7 +1321,7 @@ Error RenderingServer::mesh_create_surface_data_from_arrays(SurfaceData *r_surfa
 	if (index_array_len) {
 		List<Variant> keys;
 		p_lods.get_key_list(&keys);
-		keys.sort(); // otherwise lod levels may get skipped
+		keys.sort_custom<StringLikeVariantOrder>(); // otherwise lod levels may get skipped
```

Comment:
```suggestion
		keys.sort(); // otherwise lod levels may get skipped
```
Realized this one is just using `float` so will restore